validators/schain-upgrade.md

```diff
@@ -0,0 +1,164 @@
+ [How to back up SGX](https://skale.network/docs/documentation/sgxwallet/docs/backup-procedure)
+
+### Step 2 - TestNet Phase 3.1 Versions for SKALE Chain creation
+
+#### Important Notes - Before Update
+
+This update includes changes to docker-compose option:
+
+with the update, validators would need to support docker-compose themselves: setup and update it if necessary. 
+
+Installing docker-compose during SKALE node init is now disabled.
+
+Consensus: the build introduces BLAKE3 hash and consensus DB changes (for better performance). Should not impact update process
+
+#### Important
+
+1. Validators need to update docker-compose version on their machines to 1.27.4, because we started using new docker-compose syntax (cpu_shares functionality in particular).
+2. Validators need to remove .skale/node_data/skale.db before skale node update , because we updated db schema and until we have the migration ready we need to fast through this step by removing the folder.
+3. Resource allocation file should be re-created on all nodes - this should be done automatically during node update :
+    Why resource allocation should be updated?
+        i. In this update, we’re adding CPU and memory limits for the IMA container (this will also affect sChain container allocation)
+        ii. We’re changing the approach to estimating available memory on the machine
+        iii. The resource allocation file generation procedure and structure were revised
+
+**Note:** If DISK_MOUNTPOINT was changed in .env it’s required to do skale resources-allocation generate before update.
+
+4. Validators' Ledger devices: make sure the contract_data is ALLOWED (this can be set to Not Allowed after Ledger software update)
+5. Due to Secure Enclave changes introduced for the SGX Wallet release candidate, validators will need a backup key for the update
+
+#### Use these Versions for set up
+
+**Validator CLI version**: 1.2.0-beta.1
+
+**Node CLI version**: 1.1.0-beta.3
+
+**SGX version: sgxwallet**: 1.58.7-develop.1
+
+**SKALE Manager version**: 1.6.0-develop.7
+
+**Skaled version**:  1.46-develop.63
+
+**Skale Admin version**: 0.17.0-beta.1
+
+**Transaction Manager version**: 0.5.0-beta.1
+
+**Skale Sla version**: 1.0.0-develop.0
+
+**Skale Bounty version**: 0.9.3-develop.0
+
+#### Step 2.1 Update SGX
+
+ 1. Find your SGX back up key from the previous set up
+ 2. From sgx folder do : docker-compose down 
+ 3. Perform `git pull`
+ 4. Check-out to sgx version tag: `git checkout tags/1.58.7-develop.1`
+ 5. Make sure `image` is skalenetwork/sgxwallet:1.58.7-develop.1 in docker-compose:
+ 6. [Recover from back up] (https://skale.network/docs/documentation/sgxwallet/docs/backup-procedure)
+ 
+#### Step 2.2 Install SKALE Node CLI
+
+#### Download the SKALE Node CLI binary
+
+Make sure th `VERSION_NUM` is the 1.1.0-beta.3
+For `RELEASE` parameter use the `develop` CLI versions use `develop` , for the `beta` CLI versions use `beta` , for the `stable` CLI versions use `stable`
```

Comment:
Where is release parameter in the following command ?

Reply:
yeah, it's an old comment for the digital ocean. We can remove that line.